### DISCUSSION TITLE
In the song carousel, portions of the right click menu can be hidden by the osu cookie

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#28045] [Ideas] [answered]
When you right click a song or map entry in the song select screen, a menu is open. opening this menu in certain positions can cause it to be occluded by the osu! cookie in the bottom right, making it impossible to click the items blocked by it. 

![image](https://github.com/ppy/osu/assets/76608775/12aea85e-9516-4221-8f75-860a676999f3)

![image](https://github.com/ppy/osu/assets/76608775/54154c74-3029-45cf-9de0-0a4c01a38d7e)

**Comment by Joehuu** [comment, marked as answer]
This is similar to / already tracked as an issue in https://github.com/ppy/osu/issues/21602.
